containerstats-pipeline.md

```diff
@@ -19,20 +19,20 @@ available at `/model/containerstats/quantiles`.
 ## Behavior
 
 The pipeline builds 24 hour "windows" of data. It only builds complete windows,
-e.g. if now is `2003-08-23T08:30:00Z` the pipeline will only build up to the
+e.g. if now is `2003-08-23T08:30:00Z`, the pipeline will only build up to the
 window from `2003-08-22T00:00:00Z` to `2003-08-23T00:00:00Z`.
 
 The pipeline will return an error response if a requested time range of data
-contains any windows (24 hour chunks) are "expected" (should be in the store)
-but not "available" (the pipeline has not yet built and loaded a complete set of
+contains any windows (24 hour chunks) are expected (should be in the store)
+but not available (the pipeline has not yet built and loaded a complete set of
```

Comment:
@bstuder99 the "expected" and "available" were quoted to highlight them because they match nomenclature in the code, log messages, and error messages for this feature set. What's the best way to represent that?

Reply:
Gotcha, I just assumed the quotations were meant to introduce new terminology, but figured the following parentheses clarify these terms perfectly and so the quotes were redundant. If these terms appear in log/error messages, we should just format them in italics.

Reply:
Thanks; italicized.